dev-libs/libical/libical-2.1.0~.recipe

```diff
@@ -0,0 +1,83 @@
+SUMMARY="An implementation of iCalendar protocols and data formats"
```

Comment:
Is the ~ at the end of the package name intentional? It looks like a strange version pattern.